src/main/scripts/org/reaktivity/specification/nukleus/socks/streams/forward/client.connect.send.data/client.rpt

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright 2016-2017 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+property newClientAcceptRef ${nuklei:newReferenceId()} # external scope
+
+connect await ROUTED_CLIENT
+        "nukleus://socks/streams/source"
+  option nukleus:route ${newClientAcceptRef}
+  option nukleus:window 65536
+  option nukleus:transmission "duplex"
+
+# Client forward mode DOES NOT enforce target Nukleus to be TCP
```

Comment:
I think this is false for SOCKS RFC client.  The target needs to be TCP or UDP (no UDP nukelus yet).  The response will have the tcp extensions with the local and remote addresses.  I.E. read nukleus ext.  This is needed for the ephemeral port on the SOCKS CONNECT in https://github.com/reaktivity/nukleus-socks.spec/pull/1/files#diff-de9f689af65764d57451a65b210fe493 right?